test/e2e/manifest/manifest.go

```diff
@@ -35,7 +40,7 @@ func PodFromManifest(filename string) (*v1.Pod, error) {
 		return nil, err
 	}
 
-	json, err := utilyaml.ToJSON(data)
+	json, err := yamlutil.ToJSON(data)
```

Comment:
was this rename needed?

Reply:
i prefer to yamlutil , i think helpers usually ends with util .
:smile:

Reply:
util/yaml  -> utilyaml 
I know the reason , revert this change, make it much clear for this pr.

Reply:
yes, preferably the PR should do one understandable change. if there are multiple changes please add them in separate commits and explain all commits in the PR description.